### PR TITLE
Add like-on-github extension

### PR DESCRIPTION
**Pull request by Idnan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -392,6 +393,15 @@ Allows you to toggle between the normal GitHub contribution chart and an isometr
 
 A Chrome extension for infinite scrolling on GitHub's commit pages.
 
+## [Like On Github](https://github.com/Idnan/like-on-github)
+<a href="https://chrome.google.com/webstore/detail/lovely-forks/ialbpcipalajnakfondkflpkagbkdoib"><img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" width="24" /></a>
```
Comment (stefanbuck) [inline README.md]: URL is wrong

File: README.md [hunk 2]
```diff
@@ -392,6 +393,15 @@ Allows you to toggle between the normal GitHub contribution chart and an isometr
...
+
+Chrome extension that lets you save links to a Github repository
```
Comment (stefanbuck) [inline README.md]: Can you explain it a little be more in depth what the extension does. That would be helpful. 

Btw love this idea, well done 👍

**Comment by zeke** [comment]
This is a cool concept. So it's kind of like a bookmarking tool that uses GitHub as a data store?

**Comment by stefanbuck** [comment]
Exactly @zeke

**Comment by zeke** [comment]
tenor

**Comment by Idnan** [comment]
@zeke @stefanbuck Thanks.

**Pull request merged**